recipes-debian/avahi/avahi-ui_debian.bb

```diff
@@ -0,0 +1,53 @@
+require ${COREBASE}/meta/recipes-connectivity/avahi/avahi.inc
```

Comment:
I think the commit title should be "avahi-ui: Add avahi-ui package".
Or you could include this change in the previous commit ("avahi: Add avahi package").